### PR TITLE
add geom shapes and heat balance module

### PR DESCRIPTION
**Pull request by urtzienriquez**
In this PR I have added some shape constructors and fixed some typos. 

I have also substituted ectotherm.jl (all lowercase) with heat_balance.jl – it was giving me problems due to case sensitive/non-sensitive OS and VSCode...

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,14 +1,3 @@
...
-version = "0.1.0"
-
-[compat]
+[deps]
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
-julia = "1"
-
-[extras]
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```
Comment (rafaqz) [inline Project.toml:11]: Why remove these? Its needed for defining any tests.

File: src/GEOM.jl [hunk 2]
```diff
@@ -132,17 +182,72 @@ function sil_area_of_ellipsoid(a, b, c, θ)
...
+"""
+struct Lizard{M,D} <: Shape
+    mass::M
+    density::D
+end
+
+function geometry(shape::Lizard, ::Naked)
+    volume = shape.mass / shape.density
+    length = volume^(1 / 3)
+    mass_g = Unitful.ustrip(Unitful.uconvert(u"g", shape.mass))
```
Comment (rafaqz) [inline src/GEOM.jl:235]: Whats the reason for using `ustrip`

**Review by rafaqz — COMMENTED** [review]
Nice, looks pretty clean and easy to add shapes.

Mostly of my comments are just house-keeping details. 

The only technical/modelling question is the reason for convert and stripping units, rather than letting them flow through the models?

File: src/GEOM.jl (hunk 2 above)
Comment (mrke) [reply src/GEOM.jl:235]: empirical functions like regressions tend to mess up the units - I've had to do this elsewhere in the code too

Comment (rafaqz) [reply src/GEOM.jl:235]: Ahh of course. One option is to put he ustrip right where the empirical math happens, which is also a nice marker of where things are a bit less mechanistic

File: Project.toml [hunk 3]
```diff
@@ -1,14 +1,3 @@
-name = "Ectotherm"
-uuid = "b296eefe-8ffd-498e-9f4f-4dd5a85c833d"
-authors = ["BiophysicalEcology"]
-version = "0.1.0"
-
-[compat]
+[deps]
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```
[the remark this answers is not in the document]
Comment (urtzienriquez) [reply Project.toml]: Ok, thanks! I have removed it

File: Project.toml (hunk 1 above)
Comment (urtzienriquez) [reply Project.toml:11]: I have put them back now

File: src/GEOM.jl (hunk 2 above)
Comment (urtzienriquez) [reply src/GEOM.jl]: Yep, that was the reason. I have moved ustrip to the actual regression equations. We could also do the uconvert in the same line, but it makes the line too long. Just an stylistic comment... what do you think?

File: src/heat_balance.jl [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+# Heat balance
+
+
+function heat_balance() end
+
+heat_balance(b::AbstractBody, parameters, environment) = begin
+Qrad_in = radin()
```
[the remark this answers is not in the document]
Comment (urtzienriquez) [reply src/heat_balance.jl:8]: Yes. Should I remove it from this PR?

File: Manifest.toml [hunk 5]
```diff
@@ -1,2 +1,202 @@
 # This file is machine-generated - editing it directly is not advised
 
+julia_version = "1.8.5"
```
[the remark this answers is not in the document]
Comment (urtzienriquez) [reply Manifest.toml]: I've removed it

File: src/heat_balance.jl (hunk 4 above)
[the remark this answers is not in the document]
Comment (mrke) [reply src/heat_balance.jl:8]: might as well keep it there so we can build on it

**Pull request merged**